src/test/test_dash.cpp

```diff
@@ -54,6 +55,8 @@ BasicTestingSetup::BasicTestingSetup(const std::string& chainName)
     InitScriptExecutionCache();
     CCoinJoin::InitStandardDenominations();
     fCheckBlockIndex = true;
+    // Make sure CreateAndProcessBlock() support building <deployment_name> blocks before activating it in these tests.
+    //gArgs.ForceSetArg("-vbparams", strprintf("deployment_name:0:%d", (int64_t)Consensus::BIP9Deployment::NO_TIMEOUT));
```

Comment:
Why'd we comment this out? Can you add a comment explaining why it was commented out?

Reply:
This is simply a placeholder for <deployment_name> deployment that we don't test yet but might want to do so in the future. In the original PR it's used for segwit (which we don't have).